lib/mixpanel-node.js

```diff
@@ -132,6 +132,15 @@ var create_client = function(token, config) {
         metrics.send_request(endpoint, data, callback);
     };
 
+    var parse_time = function(time) {
+        if (time === void 0) {
```

Comment:
I'm no big fan of the `void 0` style, but didn't want to refactor just for the heck of it while extracting this code.